<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>231</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(73, 255, 228);</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TRASLACION</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>70</y>
      <width>43</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton3">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>170</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>v</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton4">
    <property name="geometry">
     <rect>
      <x>91</x>
      <y>120</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>^</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Boton2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>REINICIAR</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>240</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ESCALAR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ESCALASION</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>380</y>
      <width>62</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PFX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>PFY</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ESCALACION PUNTO FIJO</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRotar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Rotar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bodoni MT Condensed</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ROTACIÃ“N</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
    <property name="geometry">
     <rect>
      <x>71</x>
      <y>500</y>
      <width>71</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>191</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mover Gato</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Animacion1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>570</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Animacion2</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
